### DISCUSSION TITLE
Combining of mouse and keyboard

### DISCUSSION THREAD

**Discussion by deepika2802** [Vivekananda-S/Virtual_Key_Board#7] [Ideas]
As of now we need to run 2 processes to make mouse and keyboard work. It will be good if we make mouse and keyboard to run at a time.

**Comment by Vivekananda-S** [comment]
Yah..! It will be better we combine both those processes

**Comment by Vivekananda-S** [comment]
> As of now we need to run 2 processes to make mouse and keyboard work. It will be good if we make mouse and keyboard to run at a time.

Combining of mouse with keyboard is done --> IN combining mouse with keyboard in main.py path and its working fine.
But now we have a problem in returning back form mouse mode to keyboard mode
--> **Is there any module or function in autopy that can help us identify that we are typing field and notify**.(as putting another button in mouse mode will leads to another ton of problems)
-->If there is any modules which can help us..... comment back to this reply

**Comment by Shravani-646** [comment]
Try to combine both mouse and keyboard and vice-versa, like the process which opens the keyboard tab, should be executed as soon as we close the mouse tab.
